app/views/layouts/design_system.html.erb

```diff
@@ -50,7 +50,7 @@
         </div>
       <% end %>
 
-      <% if yield(:title).present? %>
+      <% if yield(:title).present? && yield(:hide_title).blank? %>
```

Comment:
I can't see why we need to stop rendering the page title. This might just need a bit more explanation.

Reply:
It's because if we don't, we end up doubling up the title with the use of the radio component. If we leave out `content_for :title, "title goes here"`, the HTML `title` attribute then gets munged, which could hurt accessibility

Reply:
Okay I think I've found the problem this is trying to solve.  
![Screenshot 2024-08-14 at 11 55 01](https://github.com/user-attachments/assets/465e4c8d-5ce6-40b4-abc1-f669c262154c)

Introducing a bit of behaviour into the main layout just to serve one of our pages felt like too much of a reach. I had another look and think we can leverage the existing `page_title` context. I'll give this a go in a new commit.

Reply:
@Harriethw I've added a new commit for this so I think it'd be good to get your approval here too. After my commit here's what we've got. We don't lose the page title nor the body title and we're able to avoid changing the global layout.
 
![Screenshot 2024-08-14 at 12 20 17](https://github.com/user-attachments/assets/55664edd-2564-4268-a531-02d7e0763e4b)

Reply:
looks good to me - makes sense!